From 9bc065e4f058931eee2315c975ec70612f0461ba Mon Sep 17 00:00:00 2001
From: Eric Sauvageau <merlin@asuswrt-merlin.net>
Date: Tue, 25 Aug 2026 23:33:28 -0400
Subject: [PATCH] openssl35: enable parallel compile

On my 26-threads dev VM this reduces compile time
from 2m55s to 0m15s.

(cherry picked from commit 0ad1504e512995d9aefa27643b04fa04c912e3e4)
---
 release/src/router/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/release/src/router/Makefile b/release/src/router/Makefile
index 5414ac3596..4cc727f4ff 100644
--- a/release/src/router/Makefile
+++ b/release/src/router/Makefile
@@ -3831,7 +3831,7 @@ openssl-3.5/Makefile:
 	@touch $@
 
 openssl-3.5: openssl-3.5/Makefile
-	$(MAKE) -C openssl-3.5 && $(MAKE) $@-stage
+	$(MAKE) -C openssl-3.5 $(PARALLEL_BUILD) && $(MAKE) $@-stage
 
 openssl-3.5-clean:
 	[ ! -f openssl-3.5/Makefile ] || $(MAKE) -C openssl-3.5 clean
-- 
2.25.1

